emperor/core.py

```diff
@@ -278,6 +291,31 @@ def make_emperor(self, standalone=False):
         # avoid unicode strings
         coord_ids = list(map(str, self.mf.index.tolist()))
 
+        # TODO: This will be removed once the custom axes creation is moved to
+        # the graphical user interface i.e. to the Axes tab.
+        if custom_axes:
```

Comment:
I'm not sure if I follow, do you mean that you will be able to create N custom axes and then select to only show, up to 2 custom axes?